codalab/lib/worksheet_util.py

```diff
@@ -617,11 +618,11 @@ def raise_genpath_usage_error():
 
             for item_index, bundle_info in bundle_infos:
                 if is_missing(bundle_info):
-                    interpreted_items.append({
-                        'mode': TYPE_MARKUP,
-                        'interpreted': 'ERROR: cannot access bundle',
-                        'properties': {},
-                    })
+                    interpreted_items.append(
+                        MarkupBlockSchema().load({
+                            'id': len(interpreted_items),
+                            "text": "ERROR: cannot access bundle",
```

Comment:
be consistent: use single quotes

Reply:
Oops @ being consistent; I thought we supposed to be using double quotes throughout the codebase?